src/components/preview/preview.stories.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { boolean, text, number } from '@storybook/addon-knobs';
+import Preview from './preview';
+import info from './documentation/info';
```

Comment:
There is `index.js` file missing, it will allows you to use import straight from `./documetation` folder

Reply:
index.js file added